libraries/core-react/src/components/EdsProvider/EdsProvider.stories.mdx

```diff
@@ -0,0 +1,25 @@
+import { Meta, Story, Canvas, ArgsTable } from '@storybook/addon-docs'
+import styled from 'styled-components'
+import { EdsProvider } from '../..'
+import * as stories from './EdsProvider.stories.tsx'
+
+<Meta title="Components/EdsProvider"
+  component={EdsProvider}
+  args={{  density: 'comfortable' }} />
+
+export const Abbr = styled.abbr`
+  cursor: help;
+`
+
+# EdsProvider
+
+Use the `<EdsProvider />` to provide users with the option of compressing the user interface to see more information at the same time without having to scroll.
```

Comment:
We have not talked to much on how we formulate our stories in Storybook, but I think this is to much text to have as an intro to the component, i felt like being hit by a wall of text and had to strive to read through it.

This feels like more of an explanation of _why_ we have a density property in EDS and not just the components, but ofcourse, there are other reasons for why this is written here not instead 🙈

Maybe the intro could be short and consice that the `EdsProvdier` is a provider for setting "settings" for the eds react components, and then have a new heading with "Accessibility" (which we should have on all our stories in the future imho) where we explain the importancy of switcher for `density`